workflow/client.go

```diff
@@ -31,6 +31,11 @@ type Client struct {
 	taskHubClient *durabletaskclient.TaskHubGrpcClient
 }
 
+type WorkflowIDReusePolicy struct {
+	OperationStatus []api.OrchestrationStatus
+	Action          api.CreateOrchestrationAction
```

Comment:
Could these types be defined/wrapped as part of the go-sdk and parsed in the options function below?